core/server/services/url/Resources.js

```diff
@@ -111,6 +119,13 @@ class Resources {
             });
     }
 
+    _fetchSingle(resourceConfig, id) {
+        let modelOptions = _.cloneDeep(resourceConfig.modelOptions);
+        modelOptions.id = id;
+
+        return models.Base.Model.raw_knex.fetchAll(modelOptions);
+    }
+
     _onResourceAdded(type, model) {
```

Comment:
Don't we need the same handling for adding resources?

Reply:
I'd suggest in this PR, because it would cause the same bug.

Reply:
Gotten rid of even more code in `_onResourceAdded` :cowboy_hat_face:

Reply:
yeeey 🤩